pkg/kubelet/container/runtime.go

```diff
@@ -617,6 +619,19 @@ func (c *RuntimeCondition) String() string {
 	return fmt.Sprintf("%s=%t reason:%s message:%s", c.Type, c.Status, c.Reason, c.Message)
 }
 
+// RuntimeFeatures contains the set of features implemented by the runtime
+type RuntimeFeatures struct {
+	SupplementalGroupsPolicy bool
+}
+
+// String formats the runtime condition into human readable string.
```

Comment:
into a human

Reply:
fixed in f46ecf5648917e007f531e91865be2d70f2c9ea6